java/com/google/location/lbs/activity/audioset/dataprocessing/CsvParser.java

```diff
@@ -0,0 +1,244 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Arrays;
+
+/**
+ * CSV Class
+ *
+ * <p>CSV is a class for the reading of a dataset in a csv format.
+ *
+ * @author Carver Forbes
+ * @version 0.1
+ */
+public class CsvParser {
+
+  private File file;
+  private ArrayList<String> ytidList;
+  private HashMap<String, IncompleteExample.Example> idToProtoMap;
+  private HashMap<String, String> idToUrlMap;
+
+  private final boolean debug = true;
```

Comment:
i'd suggest using logger instead but i'm not sure what kind of option is available externally. probably we can discuss

Reply:
Ah. A logger is a good idea. Let me see what external options are available.